### PR TITLE
Add inputs and upd workflow

### PR DESCRIPTION
**Pull request by deemp**
- Added inputs to delete by `created_at` or `last_accessed_at` time
- Updated workflow to build and then run the `purge-cache` action

Seems to work OK
https://github.com/deemp/purge-cache/actions/runs/5597514115/jobs/10235963061#step:6:19

UPD1:
I guess it can't delete caches because it's a PR.
https://github.com/actions/labeler/issues/12

UPD2:
Set `permissions.actions: write` and it deleted caches
https://github.com/deemp/purge-cache/actions/runs/5597825739/jobs/10236695224#step:6:18

UPD3:
Also OK with two inputs
https://github.com/deemp/purge-cache/actions/runs/5598490016/jobs/10238195352#step:6:9

### CODE REVIEW

**Comment by deemp** [comment]
@Arjan-Zuidema, please, have a look :pray:

File: src/index.ts [hunk 1]
```diff
@@ -44,7 +61,12 @@ async function run() {
       const cacheDate = new Date(cache.last_accessed_at);
```
Comment (Arjan-Zuidema) [inline src/index.ts]: You are always checking against `last_accessed_at`, shouldn't that be dynamic based on `created` and `accessed`?

Comment (deemp) [reply src/index.ts]: Oh, yes. Fixed that

Comment (Arjan-Zuidema) [reply src/index.ts]: You just changed the logging. Shouldn't it be something like this?

```js
if(created && accessed) {
  // Check created and/or accessed
} else if (created) {
  // Check created
} else if(accessed) {
  // Check accessed
}
```

**Review by Arjan-Zuidema — CHANGES_REQUESTED** [review]

**Comment by deemp** [comment]
@Arjan-Zuidema, you probably saw outdated changes

**Review by Arjan-Zuidema — APPROVED** [review]

**Pull request merged**